file.c

```diff
@@ -232,6 +231,15 @@ int kibosh_read(const char *path UNUSED, char *buf, size_t size, off_t offset,
     // TODO: mask EINTR?
     uid = fuse_get_context()->uid;
     ret = pread(file->fd, buf, size, offset);
+
+    // clear page cache
+    int sret = system("sudo sh -c \"echo 1 > /proc/sys/vm/drop_caches\"");
```

Comment:
Why do we do this here? A comment would be useful.

Reply:
I experimented with cleaning the cache from read calls, but the result was not very predictable. The cache was not cleared after read call returned unless we spawn a background process here. So I switched back to having one background process spawned by main to clear the cache. I changed the interval to every half second.